nitro-protocol/contracts/ForceMove.sol

```diff
@@ -134,22 +127,22 @@ contract ForceMove is IForceMove, StatusManager {
         bytes32 challengeStateHash = _hashState(
             channelId,
             variablePartAB[0].appData,
-            variablePartAB[0].outcome,
+            Outcome.encodeExit(variablePartAB[0].outcome),
             turnNumRecord,
-            isFinalAB[0]
+            variablePartAB[0].isFinal
         );
 
         bytes32 responseStateHash = _hashState(
             channelId,
             variablePartAB[1].appData,
-            variablePartAB[1].outcome,
+            Outcome.encodeExit(variablePartAB[1].outcome),
             turnNumRecord + 1,
-            isFinalAB[1]
+            variablePartAB[1].isFinal
         );
 
         // checks
 
-        bytes32 challengeOutcomeHash = keccak256(variablePartAB[0].outcome);
+        bytes32 challengeOutcomeHash = keccak256(Outcome.encodeExit(variablePartAB[0].outcome));
```

Comment:
Outside the scope of this PR: might be nice to have a `hashOutcome` helper?

Reply:
Good idea.